src/models/medicamento.js

```diff
@@ -1,7 +1,20 @@
 const sequelize = require('sequelize')
 
 module.exports = db => db.define('medicamento', {
+  id: {
+    type: sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+  },
   nome: sequelize.STRING,
   via: sequelize.STRING,
-  formaFarmaceutica: sequelize.STRING
-})
\ No newline at end of file
+  formaFarmaceutica: sequelize.STRING,
+  intervalo: sequelize.STRING,
+  dosagem: sequelize.STRING,
+  validade: sequelize.STRING,
+  lote: sequelize.STRING,
+  tipo: sequelize.STRING
+},
+{
```

Comment:
Parece que está acontecendo uma integração promíscua de código (branches integrando com outras branches que não são a `master` ou `ctrl + c` / `ctrl + v` entre branches). Isso dificulta bastante o desenvolvimento e os mesmos problemas aparecem inúmeras vezes. Neste caso, a [configuração do atributo `id`](https://github.com/aceleradora-TW/prescare/pull/27#discussion_r192807545) e do uso de `underscore` são problemas que já surgiram em uma outra PR e que estão aqui novamente. Parece que as mesmas coisas estão sendo desenvolvidas em PRs diferentes, tomem bastante cuidado com isso. :slightly_smiling_face: 

Minha sugestão seria: 
- PRs menores
- Ter branches que se integram somente com a `master` e nenhuma outra branch
- Não fazer a mesma coisa em PRs diferentes